### ISSUE TITLE
How to compare FADU output with HTSeq Output

### ISSUE THREAD

**Issue by EthanKhew** [IGS/FADU#10]
Hi. I was using HTSeq-Count to count the gene expression of a prokaryote RNA-Seq data and came across FADU. I have read the paper and would like to switch my expression analysis with FADU. 

Here, I would like to ask how can I compare my HTSeq and FADU results as you performed in the paper. 
Is there a specific approach or statistical analysis used to compare the sensitivity and accuracy of each gene-counting software?? This is to support my software switch when questioned. 

Best
- C.Y. Khew (Ethan)

**Comment by adkinsrs** [comment]
Hello @EthanKhew,

The primary author @matt-chung describes all of his analyses inside of this directory

https://github.com/IGS/FADU/tree/master/analysis

Hopefully this answers what you were looking for

**Comment by EthanKhew** [comment]
Hi @adkinsrs , 

Thanks for clarifying and pointing out what I have overlooked. 
As detailed here (https://github.com/IGS/FADU/tree/master/analysis#assess-rna-quantification-tool-performance-using-actual-rna-seq-data), the author @matt-chung conducted his analyses with unstranded and reverse stranded RNA-Seq data, whereas I am dealing with all unstranded library RNA-Seq data. At the step of preparing BAM file for more specific analyses, he calculated the depth of the BAM files and the information obtained is then proceeded with quantifying the performance of FADU vs other tools in R tools, which is what I intend to perform. 

How should I proceed with this step? Should I perform the strandedness step [ ``split_bam_by_strand.sh`` ] just as detailed by the primary author, and where can I download the .sh file? Or should  I ignore the strandedness separation step, skip to the ``samtools depth`` analysis, and proceed with the performance analyses as desired? Thanks in advance

Best, 
C.Y.Khew (Ethan)
